static/src/stylesheets/module/_adslot.scss

```diff
@@ -128,6 +128,22 @@
         }
     }
 }
+
+.large-top {
+    margin: 0 auto;
+    text-align: center;
+    z-index: 0;
+
+    .ad-slot__label {
+        width: 970px; // Size of the large banner 970x250
+        margin: 0 auto;
+    }
+
+    @include mq($until: desktop) {
+        display: none !important;
```

Comment:
I understand why we need this, but the fact we do so says something about our existing style code.